### DISCUSSION TITLE
share by email

### DISCUSSION THREAD

**Discussion by huhas12** [FreshRSS/FreshRSS#4919] [Q&A]
Hello, 

How can I change the sender when I share by email? I did a test share by email and the sender appears as "root@lockalhost" ?

**Comment by huhas12** [comment]
Changed 'from' => 'root@localhost' to my email address, tested share by email and it's showing same sender = from : "root@localhost"

**Comment by math-GH** [comment]
Could you please share a screenshot of your sharing settings?
